cirq/aqt/aqt_device.py

```diff
@@ -61,7 +66,16 @@ def __init__(self):
         self.noise_op_dict = get_default_noise_dict()
 
     def noisy_moment(self, moment: ops.Moment,
-                     system_qubits: Sequence[ops.Qid]):
+                     system_qubits: Sequence[ops.Qid]) -> List[ops.Operation]:
+        """Returns a list of noisy moments. The model incluldes
```

Comment:
nit: `includes` typo

Reply:
@pschindler This typo is not fixed.

Reply:
Oops. I was reading an outdated version of the file.